prereqs_linux.sh

```diff
@@ -20,6 +20,14 @@ else
     sudo apt-get install python3.6
 fi
 
+# pipenv
+echo "Checking for pipenv"
+if python -c "import pipenv" &> /dev/null; then
```

Comment:
In Linux systems with ubuntu, both python 2 and 3 are installed. 
`python` usually resolute to python2 and python3  is for python 3. 
We need to check which python version are we checking and then accordingly install it using pip